bot/code_coverage_bot/taskcluster.py

```diff
@@ -17,23 +17,8 @@
 NAME_PARTS_TO_SKIP = ("opt", "debug", "e10s", "1proc")
 
 
-def get_task(branch, revision, platform):
-    if platform == "linux":
-        platform_name = "linux64-ccov-opt"
-        product = "firefox"
-    elif platform == "windows":
-        platform_name = "win64-ccov-opt"
-        product = "firefox"
-    elif platform == "android-test":
-        platform_name = "android-test-ccov"
-        product = "mobile"
-    elif platform == "android-emulator":
-        platform_name = "android-api-16-ccov-debug"
-        product = "mobile"
-    else:
-        raise Exception(f"Unsupported platform: {platform}")
-
-    route = f"gecko.v2.{branch}.revision.{revision}.{product}.{platform_name}"
+def get_task(branch, revision):
```

Comment:
I would rename to `get_decision_task`, especially as you use a `decision_task_id` above

Reply:
Good idea, done!